cmd/controller-manager/app/controller-manager.go

```diff
@@ -245,6 +256,10 @@ func setInt(target *int, defaultValue int) {
 
 func setDefaultFederationConfig(fedConfig *corev1a1.FederationConfig) {
 	spec := &fedConfig.Spec
+	defaultValue := os.Getenv("DEFAULT-SCOPE")
```

Comment:
This line duplicates 106, effectively ensuring that validation of an environment variable is separate from its use.  As per my comment above, I think it would be preferable to retrieve and validate the scope inside the conditional that requires setting the scope, so this line can simply be removed.  For future, consider that separating the validation and use of an environment variable risks the possibility of the env var changing in the meantime (env vars are mutable).